Common/src/main/java/dk/gruppeseks/bodtrd/common/data/entityelements/View.java

```diff
@@ -13,6 +13,7 @@
 {
     private final String _imageFile;
     private final boolean _repeatImage;
+    private float[] _shape;
```

Comment:
shouldnt this be FOV or some sort of a more descriptive name?
also, i believe this shouldnt be part of View? maybe a different entity element would make more sense in this case